### PR TITLE
docs: [TANZUSC-1163] Fixed the rolebinding yaml and added more detail…

### PR DESCRIPTION
**Pull request by ohlookadollar**
…s about the binding

Which other branches should this be merged with (if any)?
1.1

### CODE REVIEW

File: install-components.md [hunk 1]
```diff
@@ -160,10 +160,28 @@ that you plan to create the `Workload` in:
...
 
       ```console
+      tanzu rbac binding add -g <GROUP-FOR-APP-VIEWER> -n <YOUR-NAMESPACE> -r app-viewer
+      tanzu rbac binding add -g <GROUP-FOR-APP-EDITOR> -n <YOUR-NAMESPACE> -r app-editor
+      ```
+
+      Where:
+
+      - `<YOUR-NAMESPACE>` is the name that you want to use for the developer namespace
+      - `<GROUP-FOR-APP-VIEWER>` is the group from the upstream identity provider should have access to `app-viewer` resources on the current cluster
```
Comment (julia-pu) [inline install-components.md]: I think this sentence could be a little confusing. Maybe something like `<GROUP-FOR-APP-VIEWER> is the object id of the group from the upstream identity provider that would like to have access to `app-viewer` resources on the current cluster` would be a bit clearer? Same for the line below as well

File: install-components.md [hunk 2]
```diff
@@ -160,10 +160,28 @@ that you plan to create the `Workload` in:
     ```
 
 3. Give developers namespace-level access and view access to appropriate cluster-level resources by doing one of the following:
-  * Use the `tanzu auth` plug-in to grant `app-viewer` or `app-editor` roles
-  * Apply the following RBAC policy:
+
+  **Note:** In order to apply the bindings below, an adminstrator account is required. However to use the pre-defined roles, a non adminstrator account (i.e. developer account) is required.
```
Comment (julia-pu) [inline install-components.md]: `In order to apply the bindings below, an adminstrator account is required.` Is this referring to the first option (using plugin) or the second option (applying RBAC policy) that needs an administrator account, or both? 

`However to use the pre-defined roles, a non adminstrator account (i.e. developer account) is required.` this sentence is a bit confusing to me. Do you mean that only a non administrator account can be bound to the pre-defined roles? Also typo in `adminstrator` should be `administrator`

File: install-components.md [hunk 3]
```diff
@@ -174,24 +192,58 @@ that you plan to create the `Workload` in:
...
+          name: <GROUP-FOR-APP-EDITOR>
           apiGroup: rbac.authorization.k8s.io
       EOF
       ```
 
+      Where:
+
+      - `<YOUR-NAMESPACE>` is the name that you want to use for the developer namespace
+      - `<GROUP-FOR-APP-VIEWER>` is the group from the upstream identity provider should have access to `app-viewer` resources on the current cluster
+      - `<GROUP-FOR-APP-EDITOR>` is the group id from the upstream identity provider should have access to `app-editor` resources on the current cluster
```
Comment (julia-pu) [inline install-components.md]: Same as previous comment above

File: install-components.md (hunk 1 above)
Comment (ohlookadollar) [reply install-components.md]: I excluded group id, just because it may not be an id. It really depends on the idp

**Pull request merged**